test1/CellViewController.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  CellViewController.swift
+//  test1
+//
+//  Created by Nikola Blagoevski on 22.4.22.
+//
+
+import UIKit
+import SnapKit
+import SDWebImage
+
+class NikolaAwesomeLabel: UILabel{
+    override init(frame: CGRect){
+        super.init(frame: frame)
+        self.backgroundColor = .white
+        self.font = .systemFont(ofSize: 20)
+        self.textAlignment = .left
+        self.numberOfLines = 0
+        self.lineBreakMode = .byWordWrapping
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+class CellViewController: UIViewController{
+    
+
+    var person: CharacterModel
+    //let tableView = UITableView()
+    
+    init(person: CharacterModel){
+        self.person = person
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        //view.addSubview(tableView)
+//      tableView.delegate = self
+//      tableView.dataSource = self
+        makeStackView()
+    }
+    
+    lazy var image: UIImageView = {
```

Comment:
property definitions go before viewdidload and any other functions

Reply:
I moved the makeStackView() function from viewDIdLoad() to init()

Reply:
I meant to move the function body higher up in the code. won't block you on this

Reply:
Ok, i will merge then